BranchBrowser.py

```diff
@@ -525,6 +526,36 @@ def delete_branch(self):
             self.update_tree(None) # Update tree to reflect changes
         else:
             print(f"Deleting branch {branch_name} on {org_name}/{repo_name} canceled!")
+
+    def delete_branch_with_submodules(self):
+        org_name = self.org_combo.get()
+        repo_name = self.repo_combo.get()
+        selected_item = self.last_tree_item_rightclicked
+        branch_name = get_path(self.branches_tree, selected_item)
+
+        # Fetching all organization repositories
+        repo_names = self.github_client.get_organization_repos_names(org_name)
+        print(repo_names)
+
+        repos_with_branch = []
+
+        for repo_name in repo_names:
+            branches = self.github_client.get_organization_repo_branches(org_name, repo_name)
+            print(branches)
+            if branch_name not in branches:
+                continue
+            else:
+                repos_with_branch.append(repo_name)
+
+        result = DeleteWithSubmodulesDialog(self.root, self.github_client, org_name, branch_name, repos_with_branch).result

```

Comment:
Linija je suvise dugacka. Neka preporuka je da to bude 80 karaktera, ali mi unutar tima imamo dogovor da to bude 120 karaktera. Nije dobro da linije budu toliko dugacke jer umanjuju preglednost i navigaciju kroz sam code. Dodatni benefit je ako koristis side-by-side view u pyCharm-u, onda u mozes lepo da pratis code na 2 mesta a da ne otvaras ili da se ne switch-ujes na drugi file/klasu.